packages/inter-protocol/test/smartWallet/test-psm-integration.js

```diff
@@ -311,22 +281,35 @@ test('govern offerFilter', async t => {
   // 44 tested above
   t.is(currentState.offerToUsedInvitation[46].value[0].description, 'Voter0');
 
+  // Call for a vote ////////////////////////////////
```

Comment:
consider `t.log('Call for a vote')`?

Reply:
I'm really on the fence about that. Is it useful log output? It won't interleave with debugging output.

I'm opting to punt on this change until there's a clear best practice.

Reply:
> Is it useful log output?

I like it when the `t.log()` output tells a story...

> It won't interleave with debugging output.

I think it does with failures of `t.is()` and `t.deepEqual()` and such. But yeah, it's out of sync with `console.log()`.

Punting is fair. Thanks for considering it.